app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new()
+  end
+
+  def create
+    @user = User.new(user_params)
+    
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+  def user_params
```

Comment:
the `user_params` method should be added at the end of the class and also should be set as `private`